pkgs/os-specific/linux/acpi-call/default.nix

```diff
@@ -1,43 +1,31 @@
-{ stdenv, fetchgit, fetchpatch, kernel }:
+{ stdenv, fetchFromGitHub, fetchpatch, kernel }:
 
 stdenv.mkDerivation {
-  name = "acpi-call-${kernel.version}";
+  name = "acpi-call-2020-04-07-${kernel.version}";
 
-  src = fetchgit {
-    url = "git://github.com/mkottman/acpi_call.git";
-    rev = "ac67445bc75ec4fcf46ceb195fb84d74ad350d51";
-    sha256 = "0jl19irz9x9pxab2qp4z8c3jijv2m30zhmnzi6ygbrisqqlg4c75";
+  src = fetchFromGitHub {
+    owner = "Mic92";
```

Comment:
PR description says «nix-community», is this a before-transfer version?

Reply:
Good catch. Let me fix it!